src/pages/Delegate/index.tsx

```diff
@@ -21,14 +21,43 @@ import { Title } from '@/components/ui/title'
 import { DelegateCard } from '@/components/DelegateCard'
 
 export const Delegate = () => {
-  const { api, assetInfo } = useNetwork()
-  const { address } = useParams()
-  const { getConvictionLockTimeDisplay } = useLocks()
+  const { api, assetInfo, selectNetwork } = useNetwork()
+  const { address, incomingNetwork } = useParams()
+  const { selectedAccount } = useAccounts()
+  const getDelegateTx = useGetDelegateTx()
+  const { getConvictionLockTimeDisplay, refreshLocks } = useLocks()
+  const getSubscriptionCallBack = useGetSigningCallback()
+  const navigate = useNavigate()
+  const { search } = useLocation()
+  const [newAddress, setNewAddress] = useState<string>()
```

Comment:
I'm surprised this passes compilation since it's undefined.
Can we call this `validAddress`? since it's not really new.
```suggestion
  const [newAddress, setNewAddress] = useState<string>('')
```

Reply:
useState does not care at init if its undefined (if it is filled on first `set...` it wont complain) - but sure

Reply:
it's not about if it cares or not, it's about the type you've set. if it's `string` but you set it to undefined, then it'll complain that string can't be undefined.